pkg/cstorvolumereplica/v1alpha1/cstorvolumereplica_test.go

```diff
@@ -0,0 +1,206 @@
+package v1alpha1
+
+import (
+	"reflect"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+)
+
+func TestGetPoolUIDs(t *testing.T) {
+	tests := map[string]struct {
+		cvrs           []cvr
+		expectedString []string
+	}{
+		//  UIDS are present
+		"Present 1": {
+			[]cvr{
+				cvr{apis.CStorVolumeReplica{ObjectMeta: metav1.ObjectMeta{Labels: map[string]string{cstorPoolUIDLabelKey: "uid1"}}}},
```

Comment:
Same comment. The table will have only primitive data types. With these code the entire test logic is not readable.

Reply:
done